docs/overview/architecture.rst

```diff
@@ -10,14 +10,70 @@ power the CommCare website.
 Each module is a collection of one or more Django applications that each contain the relevant data models, url
 mappings and view controllers, templates, and Database views necessary to provide that module’s functionality.
 
-Internal Analytics and transformation Engines
----------------------------------------------
-
-The analytics engines are used for offline processing of raw data to generate aggregated values used in reporting
-and analytics. There are a suite of components that are used which are roughly diagrammed below. This offline
-aggregation and processing is necessary to keep reports running on huge volumes of data fast.
-
-.. image:: ../images/reporting_architecture.png
+Data flow for forms and cases
+-----------------------------
+
+CommCare deals with many different types of data but the primary data that is generated by users form and case data.
+This data most often comes from a mobile device running the CommCare mobile application. It may also come from
+Web Apps, via an integration API or from a case import.
+
+The data processing for form and case data is split into two portions, synchronous processing and asynchronous
+processing.
+
+Synchronous processing
+~~~~~~~~~~~~~~~~~~~~~~
+Form and case data in CommCare ALWAYS follow the same pathway through the system regardless of whether the data
```

Comment:
Two separate suggestions here. Typo: "follow" -> "follows" is most important. Less important, emphasis on the word "ALWAYS" does not seem necessary.

```suggestion
Form and case data in CommCare always follows the same pathway through the system regardless of whether the data
```